test/Mono.Linker.Tests.Cases/References/AssemblyOnlyUsedByUsingWithMcs.cs

```diff
@@ -3,9 +3,12 @@
 using Mono.Linker.Tests.Cases.References.Dependencies;
 
 namespace Mono.Linker.Tests.Cases.References {
+#if NETCOREAPP
```

Comment:
Please move such tests under /Mcs

Reply:
The existing convention would be for this to go under, for example, `References/Mcs/`, and those below under `TestFramework/Mcs/`. Is that what you'd like, or are you asking for `Mcs` to be the top-level with subdirectories like `Mcs/References`, `Mcs/TestFramework`?

Reply:
I think it should be all under mcs/* that way it's clear the tests are for mcs behaviour only

Reply:
Doing this in https://github.com/mono/linker/pull/493.

Reply:
As a compromise (see discussion in https://github.com/mono/linker/pull/493), I've grouped the mcs tests together in the project file and excluded them from compilation in https://github.com/mono/linker/pull/489/commits/e239b64768549537cff339b617ba689d0deeda75 in this PR. We can always move the mcs tests around later if desired. Does that seem good to you?

Note that excluding them from `Compile` will result in messages in the output like:
```
Could not find the matching type for test case /Users/sven/src/dotnet/linker/test/Mono.Linker.Tests.Cases/Attributes/Mcs/OnlyTypeUsedInAssemblyIsTypeOnAttributePropertyOnType.cs.  Ensure the file name and class name match
```

Reply:
That output is to help identify when a test case is named incorrectly.  So we really do not want to introduce these.

I would recommend we leave it how you had it originally.  Add the #if NETCOREAPP w/ ignores and let's worry about this later if it becomes an annoyance.  @marek-safar Are you OK with that?